apps/meteor/ee/server/api/sessions.ts

```diff
@@ -105,7 +105,7 @@ API.v1.addRoute(
 
 				search.push(
 					...(await Users.findActiveByUsernameOrNameRegexWithExceptionsAndConditions<Pick<IUser, '_id'>>(
-						{ $regex: filter, $options: 'i' },
+						new RegExp(filter, 'i'),
```

Comment:
Why? 👀

Reply:
Ah nvm I confused between different things.

Reply:
made sense to not mix types so much 😬  they're the same in this context any way

Reply:
But, is this function only used here? Cause changing the params would break other places if any 👀

Reply:
I'm with you on that one. But yeah, to keep the change pls find other places using the func and update accordingly